base/applications/mspaint/history.cpp

```diff
@@ -17,6 +17,8 @@ void ImageModel::NotifyImageChanged()
 {
     if (canvasWindow.IsWindow())
         canvasWindow.Invalidate(FALSE);
+    if (miniature.IsWindow())
+        miniature.Invalidate(TRUE);
```

Comment:
Is the "TRUE" parameter here used to make the miniature window redraw completely?

Reply:
This `TRUE` is for `::InvalidateRect` function's 3rd parameter `bErase`.
`FALSE` is also OK for this implementation.

Reply:
ok yes, I wanted to understand why the canvas got FALSE while the miniature got TRUE.

Reply:
Fixed.